install_ffmpeg.sh

```diff
@@ -111,7 +111,12 @@ else
   if which clang > /dev/null; then
     echo "clang detected, building with GPU support"
 
-    EXTRA_FFMPEG_FLAGS="--enable-cuda --enable-cuda-llvm --enable-cuvid --enable-nvenc --enable-decoder=h264_cuvid --enable-filter=scale_cuda --enable-encoder=h264_nvenc --enable-libtensorflow --enable-filter=lvpdnn"
+    EXTRA_FFMPEG_FLAGS="--enable-cuda --enable-cuda-llvm --enable-cuvid --enable-nvenc --enable-decoder=h264_cuvid --enable-filter=scale_cuda --enable-encoder=h264_nvenc"
+
+    if [[ $BUILD_TAGS == *"experimental"* ]]; then
+        echo "experimental tag detected, building with Tensorflow support"
+        EXTRA_FFMPEG_FLAGS="--enable-tensorflow"
```

Comment:
Think we want to add `--enable-tensorflow` to `EXTRA_FFMPEG_FLAGS` and not set the var which would overwrite its existing value yeah?

Reply:
Ah good catch, my bad.

Reply:
Fixed in https://github.com/livepeer/go-livepeer/pull/1926/commits/19d2c35db875faab4f0e058afeb86bd94685dca6